config/efm-langserver/config.yaml

```
@@ -235,6 +235,15 @@ tools:
       - '%tARNING %f:%l %m'
       - '%tRROR %f:%l %m'
 
+  rstcheck-lint: &rstcheck-lint
+    lint-command: 'rstcheck -'
+    lint-stdin: true
+    lint-formats:
+      - '%f:%l: (%tNFO/%r) %m'
+      - '%f:%l: (%tARNING/%r) %m'
+      - '%f:%l: (%tRROR/%r) %m'
+      - '%f:%l: (%tEVERE/%r) %m'
```

```suggestion
      - '%f:%l: (%tNFO/%*[A-Z0-9]) %m'
      - '%f:%l: (%tARNING/%*[A-Z0-9]) %m'
      - '%f:%l: (%tRROR/%*[A-Z0-9]) %m'
      - '%f:%l: (%tEVERE/%*[A-Z0-9]) %m'
```

```
	%r		matches the "rest" of a single-line file message %O/P/Q
	%*{conv}	any scanf non-assignable conversion
```

message `<file>:<line>: (ERROR/3) (python) foobar...` and `(%tRROR/%r)` matcher, match `(ERROR/3) (python)` as many as possible matching result.